Source/WebKit/Shared/AuxiliaryProcessExtensions/AuxiliaryProcessExtensionBridge.h

```diff
@@ -25,11 +25,13 @@
 
 #pragma once
 
+#import <xpc/xpc.h>
```

Comment:
This is now hazardous for open source builders, right?

As a bridging header, I think(?) you can safely import XPCSPI.h from here, since it's not part of the module.

Reply:
That is a good point. However, I don't think config.h is included in all cases for this header, which prevents us from including XPCSPI.h. I plan on only building this for internal builds, initially.

Reply:
I see. I guess it's fine since this is bringup work and this header is not yet part of the build, but I'd appreciate a FIXME here to help us remember it needs refactoring :)

Reply:
That's a good point, FIXME added :)